tests/integration/test_stylelint_integration.py

```
@@ -0,0 +1,133 @@
+    '"declaration-block-no-duplicate-properties": true } }'
+)
+
+
+def _stylelint_available() -> bool:
+    """Report whether the stylelint binary is runnable in this environment.
+
+    Returns:
+        True if ``stylelint --version`` succeeds via the resolved command.
+    """
+    plugin = StylelintPlugin()
+    cmd = plugin._get_executable_command(tool_name="stylelint") + ["--version"]
```

Ruff (0.15.20)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)